hack/stabilization-changes.py

```diff
@@ -2,6 +2,7 @@
 
 import codecs
 import datetime
+import dateutil.relativedelta
```

Comment:
[`dateutil`][1] is not part of Python's standard library.  I'm not going to attempt local implementations of YAML parsers or GitHub clients, and we can't avoid those for this script, so I'm fine with 3rd party libraries for those.  But I think we can get by without convenient duration formating, when Python's stdlib formatting isn't too bad:

```console
$ python -c 'import datetime; print(datetime.timedelta(seconds=876543.21))'
10 days, 3:29:03.210000
```

If we want to round to the nearest second to avoid milliseconds, I'd be fine with:

```python
version_delay = datetime.timedelta(seconds=round(now - feeder_promotion['committer-time']).total_seconds()))
```

like:

```console
$ python -c 'import datetime; print(datetime.timedelta(seconds=round(datetime.timedelta(seconds=876543.21).total_seconds())))'
10 days, 3:29:03
```

Thoughts?

[1]: https://pypi.org/project/python-dateutil/

Reply:
Right, actually we don't need `dateutil`. Reworked this in aa7cd1d6dc4c984a3df6b9b082b47361af97d093